data-prepper-core/src/test/java/com/amazon/dataprepper/server/DataPrepperServerTest.java

```diff
@@ -50,6 +58,76 @@ public void testScrapeFailure() throws IOException, InterruptedException, URISyn
         dataPrepperServer.stop();
     }
 
+    @Test
+    public void testListPipelines() throws URISyntaxException, IOException, InterruptedException {
+        setRegistry(new PrometheusRegistryMockScrape(PrometheusConfig.DEFAULT, ""));
+        final DataPrepper mockDataPrepper = Mockito.mock(DataPrepper.class);
```

Comment:
nit: Shared setup code like this which _always_ is executed could be brought up into a `@Before` method to make the tests themselves more concise.

Reply:
Yeah, it will need to be overridden in the prometheus specific tests, but I will add this in to an @Before function for the rest of the tests.